homeassistant/components/workday/strings.json

```diff
@@ -83,5 +83,48 @@
         "holiday": "Holidays"
       }
     }
+  },
+  "issues": {
+    "bad_country": {
+      "title": "Configured Country for {title} does not exist",
```

Comment:
Should we clarify this is about a workday binary sensor or that's not needed?

Reply:
Not needed as one `binary_sensor` is the sole purpose of this integration